include/ipr/impl

```diff
@@ -1049,6 +1050,7 @@ namespace ipr {
          Optional<ipr::Expr> resolution() const final;
       };
 
+      using Materialize = Classic_unary_expr<ipr::Materialize>;
```

Comment:
Is there a reason to use `Classic_unary_expr` as opposed to `Unary_expr`? I would not expect these casts to be overloaded.

Reply:
This was a copy-and-past error, will update to Unary_expr.